db/migrate/20210615091947_add_user_id_to_missions.rb

```diff
@@ -0,0 +1,12 @@
+class AddUserIdToMissions < ActiveRecord::Migration[6.0]
+  def up
+    add_column :missions, :user_id, :bigint
+    add_index :missions, :user_id
+    Mission.update_all(user_id: 1)
```

Comment:
這樣子直接塞入 `user_id: 1` 的寫法並沒有維持 data integrity （ `user.rb` 的 `dependent: destroy` 表示了 missions 一定都會有個對應的實際 user ； database 不會有無實際 user 的 missions ），僅保證已建立的 missions 的欄位 `user_id` 不為空值。

請至少寫到像是下方，保證有 user 。

```suggestion
    user = User.create(email: 'admin@task.list') if User.all.size.zero?
    Mission.update_all(user: user)
```

Reply:
最初是方向是保留一個預設 User 是不能被刪除，所以直接指定 id = 1 (同時做到最後一個管理員不能刪除)
但看完這段有思考到原本的方向的彈性很低，id = 1 的使用者就完全不可做危險動作，若有意外發生，任務全數消失的可能性就會很大